package.json

```diff
@@ -12,7 +12,7 @@
     "postinstall": "lerna bootstrap",
     "build": "gulp",
     "build-all": "gulp all",
-    "release": "lerna bootstrap; gulp; lerna publish --skip-git; gulp version;",
+    "release": "lerna bootstrap; lerna publish; gulp version; gulp",
```

Comment:
Hm, but now we basically publish before we change the version globally? and the `--exact` is missing, which is quite essential.

Reply:
See the changes in lerna.json.